### DISCUSSION TITLE
Issue with Filament Login Route: "Not Found" Error When Accessing /admin/login

### DISCUSSION THREAD

**Discussion by krummyjack** [filamentphp/filament#14163] [Help] [answered]
### Package

Panel builder

### Package Version

3.2

### How can we help you?

Hello Filament team,

I'm experiencing an issue with the installation of Filament on my Laravel project. Despite following the installation steps correctly, I encounter a "Not Found: The requested URL was not found on this server" error when trying to access the Filament login form at /admin/login. Below are the details of my environment and the steps I have taken:

Environment:

- Laravel Version: 11.22.0
- Filament Version: 3.2.x
- Livewire Version: 3.5.6
- PHP Version: 8.3.6
- Server: Apache/2.4.58 (Ubuntu)
- Database: MySQL

Steps Taken:

1. Laravel Installation: Created a Laravel project using composer create-project laravel/laravel AA_hosteleria.

2. Configuration of .env File: Ensured that APP_URL is set to http://192.168.0.107/AA_hosteleria/public.

3. Filament Installation:

- Ran composer require filament/filament:"^3.2" -W.
- Set up Filament using php artisan filament:install --panels.

4. Verification of Filament Routes:

- Confirmed that Filament routes are correctly registered using php artisan route:list | grep admin, and I see the following routes:
`GET|HEAD   admin .................................. filament.admin.pages.dashboard › Filament\Pages › Dashboard
GET|HEAD   admin/login ............................ filament.admin.auth.login › Filament\Pages › Login
POST       admin/logout ........................... filament.admin.auth.logout › Filament\Http › LogoutController
`

5. Testing Laravel Routes:

- I tested several basic Laravel routes to confirm that the server and Laravel configurations are functioning correctly. All these routes work without issues, suggesting no configuration problems with Apache or Laravel.:

`Route::get('/', function () {
    return view('welcome');
});

// Basic diagnostic route
Route::get('/test', function () {
    return 'Laravel is working correctly!';
});

// Route with parameters
Route::get('/test/{name}', function ($name) {
    return "Hello, {$name}!";
});

// Route with a view
Route::get('/view-test', function () {
    return view('welcome'); // Ensure the 'welcome' view exists in resources/views
});
`

**Issue Encountered**:

Despite the Filament routes appearing to be correctly registered and configured, when trying to access after login http://192.168.0.107/AA_hosteleria/public/admin/login, I receive the "Not Found" error.

**Additional Actions Taken**:

- Checked file permissions and server configurations, including AllowOverride All in Apache settings.
- Cleared caches and configurations multiple times using php artisan cache:clear, config:clear, route:clear, and view:clear.
- Ensured Livewire is installed and its assets are published correctly (php artisan vendor:publish --force --tag=livewire:assets).

**Additional Observations:**

- The password field in the login form was initially not obscured but was resolved after publishing Livewire assets.
- Reinstalled Filament and updated all dependencies to ensure versions are correct and compatible.

**Comment by krummyjack** [comment]
![image](https://github.com/user-attachments/assets/7fa150ea-9610-44fd-b923-d40367abf386)


![image](https://github.com/user-attachments/assets/496fd307-e7cf-4022-8e4f-b58b87c21b2f)

**Reply by krummyjack** [reply]
It seems that the problem comes from the Livewire update. It seems that it does not take the correct path of the project

![image](https://github.com/user-attachments/assets/baf8544a-b3a3-41d6-baf2-0789b9efe37f)

**Reply by krummyjack** [reply, marked as answer]
Solved add in routes/web.php adapting the name to your project:

use Livewire\Livewire;

Livewire::setScriptRoute(function ($handle) {
    return Route::get('/AA_hosteleria/public/livewire/livewire.js', $handle);
});

Livewire::setUpdateRoute(function ($handle) {
    return Route::post('/AA_hosteleria/public/livewire/update', $handle);
});

**Reply by TeyeHack** [reply]
It worked for me as well, thanks for the solution

**Reply by Mouneeb** [reply]
Thank you man, it solved the problem, I've been working on it scene yesterday.
